nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -967,6 +967,58 @@ public void supportObjectsIsNull() {
         .doTest();
   }
 
+  @Test
+  public void supportAssertThatIsNotNull_String() {
+    compilationHelper
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import java.util.Objects;",
+            "import javax.annotation.Nullable;",
+            "class Test {",
+            "  private void foo(@Nullable String s) {",
+            "    com.google.common.truth.Truth.assertThat(s).isNotNull();",
+            "    s.toString();",
+            "  }",
+            "}")
+        .doTest();
+  }
+
+  @Test
+  public void supportAssertThatIsNotNull_Object() {
+    compilationHelper
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import java.lang.Object;",
+            "import java.util.Objects;",
+            "import javax.annotation.Nullable;",
+            "class Test {",
+            "  private void foo(@Nullable Object o) {",
+            "    com.google.common.truth.Truth.assertThat(o).isNotNull();",
+            "    o.toString();",
+            "  }",
+            "}")
+        .doTest();
+  }
+
```

Comment:
After adding the flag, do add a positive test that returns an error on the same code with the flag turned off.

Reply:
Added a test with flag turned off.